packages/documentation/src/stories/components/header/components/header.markup.ts

```diff
@@ -67,14 +73,19 @@ export default html`<post-header>
             <post-list-item><a href="">Waren Ausland</a></post-list-item>
             <post-list-item><a href="">Express und Kurier</a></post-list-item>
           </post-list>
-          <post-closebutton>Schliessen</post-closebutton>
         </post-megadropdown>
       </post-list-item>
       <post-list-item>
-        <post-megadropdown-trigger> Pakete </post-megadropdown-trigger>
-        <post-megadropdown>
-          <button slot="back-button"><- Zurück</button>
-          <h2><a href="">Pakete title</a></h2>
+        <post-megadropdown-trigger for="pakete">
+          <button class="btn btn-link">Pakete</button>
```

Comment:
Please remove the button classes for all `post-megadropdown-trigger`:
```suggestion
          <button>Pakete</button>
```